docs_user/modules/proc_verifying-the-image-service-adoption.adoc

```diff
@@ -0,0 +1,128 @@
+[id="verifying-the-image-service-adoption_{context}"]
+
+= Verifying the {image_service} adoption
+
+Verify that you successfully adopted your {image_service_first_ref} to the {rhos_long} {rhos_curr_ver} deployment.
+//kgilliga: Does this procedure apply to all backends, or do some verification steps only apply to specific backends?
+
+.Procedure
+
+. Test the glance service from the {rhos_prev_long} CLI. You can compare and make sure the configuration has been correctly applied to the glance pods:
++
+----
+./os-diff cdiff --service glance -c /etc/glance/glance.conf.d/02-config.conf  -o glance_patch.yaml --frompod -p glance-api
+----
++
+If no line appears, then the configuration is correctly done.
+
+. Inspect the resulting glance pods and ensure that Ceph secrets are properly mounted:
+//kgilliga: Does this step only apply to customers who are using a Ceph backend?
++
+----
+GLANCE_POD=`oc get pod |grep glance-default-external-0 | cut -f 1 -d' '`
+oc exec -t $GLANCE_POD -c glance-api -- cat /etc/glance/glance.conf.d/02-config.conf
+
+[DEFAULT]
+enabled_backends=default_backend:rbd
+[glance_store]
+default_backend=default_backend
+[default_backend]
+rbd_store_ceph_conf=/etc/ceph/ceph.conf
+rbd_store_user=openstack
+rbd_store_pool=images
+store_description=Ceph glance store backend.
+
+oc exec -t $GLANCE_POD -c glance-api -- ls /etc/ceph
+ceph.client.openstack.keyring
+ceph.conf
+----
+
+. Check that the service is active and the endpoints are properly updated in the {OpenStackShort} CLI:
++
+----
+(openstack)$ service list | grep image
+
+| fc52dbffef36434d906eeb99adfc6186 | glance    | image        |
+
+(openstack)$ endpoint list | grep image
+
+| 569ed81064f84d4a91e0d2d807e4c1f1 | regionOne | glance       | image        | True    | internal  | http://glance-internal-openstack.apps-crc.testing   |
+| 5843fae70cba4e73b29d4aff3e8b616c | regionOne | glance       | image        | True    | public    | http://glance-public-openstack.apps-crc.testing     |
+| 709859219bc24ab9ac548eab74ad4dd5 | regionOne | glance       | image        | True    | admin     | http://glance-admin-openstack.apps-crc.testing      |
+----
+
+. Check that the images that you previously listed in the source Cloud are available in the adopted service:
++
+----
+(openstack)$ image list
++--------------------------------------+--------+--------+
+| ID                                   | Name   | Status |
++--------------------------------------+--------+--------+
+| c3158cad-d50b-452f-bec1-f250562f5c1f | cirros | active |
++--------------------------------------+--------+--------+
+----
+
+. Test that an image can be created on the adopted service:
```

Comment:
This step tests the Image service is working by uploading an image. Not sure we need it downstream.